rocAL/rocAL_pybind/amd/rocal/readers.py

```diff
@@ -22,158 +22,184 @@
-           skip_cached_images=False, stick_to_shard=False, tensor_init_bytes=1048576, device=None):
 
-    #Output
+def caffe2(path, bbox=False):
+
+    # Output
     bboxes = []
     labels = []
-    kwargs_pybind = {"source_path": path, "is_output":True}
+    kwargs_pybind = {"source_path": path, "is_output": True}
     if (bbox == True):
         Pipeline._current_pipeline._reader = "Caffe2ReaderDetection"
-        caffe2_meta_data = b.caffe2ReaderDetection(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        caffe2_meta_data = b.caffe2ReaderDetection(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     else:
         Pipeline._current_pipeline._reader = "Caffe2Reader"
-        caffe2_meta_data = b.caffe2Reader(Pipeline._current_pipeline._handle ,*(kwargs_pybind.values()))
+        caffe2_meta_data = b.caffe2Reader(Pipeline._current_pipeline._handle, *(kwargs_pybind.values()))
     if (bbox == True):
-        return (caffe2_meta_data,bboxes, labels)
+        return (caffe2_meta_data, bboxes, labels)
     else:
         return (caffe2_meta_data, labels)
 
-def video(*inputs, sequence_length, additional_decode_surfaces=2, bytes_per_sample_hint=0, channels=3,
-          dont_use_mmap=False, dtype=types.FLOAT, enable_frame_num=False,  enable_timestamps=False, file_list="",
-          file_list_frame_num=False, file_list_include_preceding_frame=False, file_root="", filenames=[],
-          image_type=types.RGB, initial_fill=1024, labels="", lazy_init=False, normalized=False, num_shards=1,
-          pad_last_batch=False, pad_sequences=False, prefetch_queue_depth=1, preserve=False, random_shuffle=False,
-          read_ahead=False, seed=-1, shard_id=0, skip_cached_images=False, skip_vfr_check=False, step=1,
-          stick_to_shard=False, stride=1, tensor_init_bytes=1048576, decoder_mode=types.SOFTWARE_DECODE,
-          device=None, name=None):
+
+def video(sequence_length, file_list_frame_num=False, file_root="", image_type=types.RGB, num_shards=1,
+          random_shuffle=False, step=1, stride=1, decoder_mode=types.SOFTWARE_DECODE):
 
     Pipeline._current_pipeline._reader = "VideoDecoder"
-    #Output
+    # Output
     videos = []
-    kwargs_pybind_reader = {"source_path": file_root,"sequence_length":sequence_length,"frame_step":step,"frame_stride":stride,"file_list_frame_num":file_list_frame_num} #VideoMetaDataReader
-    b.VideoMetaDataReader(Pipeline._current_pipeline._handle ,*(kwargs_pybind_reader.values()))
-    kwargs_pybind_decoder = {"source_path": file_root,"color_format":image_type,"decoder_mode":decoder_mode,"shard_count":num_shards,"sequence_length":sequence_length,"shuffle":random_shuffle ,"is_output":False,"loop":False, "frame_step":step,"frame_stride":stride, "file_list_frame_num":file_list_frame_num } #VideoDecoder
-
-    videos = b.VideoDecoder(Pipeline._current_pipeline._handle ,*(kwargs_pybind_decoder.values()))
+    kwargs_pybind_reader = {
```

Comment:
Should we have this in multiple lines please check

Reply:
I used a python formatter to format the files which changes the one line dict definition to multi lines